### DISCUSSION TITLE
why do .ssc files even exist?

### DISCUSSION THREAD

**Discussion by ghost** [TeamRizu/OutFox#738] [04 - Q&A] [answered]
I don't really see the point (get it?). They cut off compatibility with other pieces of sofware I'm using. They literally do the EXACT same thing as .sm files. Yet, it's pretty widley adopted. Why not keep things the same?

**Comment by Sudospective** [comment]
ssc has a lot more features over sm, including split timing, separate previews for charts, and custom time signatures. they are not exactly similar.

**Comment by Jousway** [comment, marked as answer]
https://outfox.wiki/en/dev/effects/EffectFiles-ChartSegments

**Reply by ghost** [reply]
that's cool. thanks for explaining!
